libs/sql-schema-describer/src/lib.rs

```diff
@@ -135,7 +135,7 @@ impl Table {
             .unwrap_or_else(|| panic!("Column {} not found in Table {}", name, self.name))
     }
 
-    pub fn column(&self, name: &str) -> Option<&Column> {
+    pub fn column<'a>(&'a self, name: &str) -> Option<&'a Column> {
```

Comment:
this method in isolation works, but when using it inside iterators it breaks down, we need to pin the lifetime of the column on `self` (otherwise it could be the `name` reference's lifetime)